test/check-application

```diff
@@ -1447,14 +1447,73 @@ class TestFiles(testlib.MachineCase):
         b.wait_in_text(".pf-v5-c-modal-box__title-text", "bin")
         b.wait_in_text("#edit-permissions-owner", "root")
         b.wait_in_text("#edit-permissions-group", "root")
-
-        # As normal user you cannot change user/group permissions
-        b.drop_superuser()
+        b.click("button.pf-m-link")
+        b.wait_not_present(".pf-v5-c-modal-box")
 
         b.go("/files#/?path=/home/admin")
         self.assert_last_breadcrumb("admin")
         b.wait_not_present(".pf-v5-c-empty-state")
 
+        # Enclosed folder permissions
+        change_enclosed_dir = "/home/admin/Documents"
+        m.execute(f"""
+            runuser -u admin mkdir {change_enclosed_dir}
+            runuser -u admin mkdir {change_enclosed_dir}/Finance
+            runuser -u admin mkdir {change_enclosed_dir}/Work
+            runuser -u admin chmod 750 {change_enclosed_dir}/Work
+            runuser -u admin touch {change_enclosed_dir}/Finance/numbers.txt
+            runuser -u admin chmod 777 {change_enclosed_dir}/Finance/numbers.txt
```

Comment:
This could be `script.sh` to make the assertion below easier to understand -- i.e. point out that it is executable. Similar to report.txt

Reply:
Already wrote something along those lines.